### ISSUE TITLE
Some issues to fix

### ISSUE THREAD

**Issue by scopartu** [theNewDynamic/gohugo-theme-ananke#723]
See https://github.com/gohugoio/hugoDocs/issues/2548#issuecomment-2334770313

**Comment by davidsneighbour** [comment]
As noted in that original issue, these warnings are addressed. I will create a release around the weekend, until then you can either pull the updates manually in your submodule or if you chose the module way run `hugo mod get -u ./...` to load the updates. I'll close this issue when the new milestone is released.

**Comment by davidsneighbour** [comment]
Fixed in v2.10.0

**Comment by kilejaslan** [comment]
```
WARN  deprecated: .Site.Social was deprecated in Hugo v0.124.0 and will be removed in a future release. Use .Site.Params instead.
```
